sdk/app_cpu1/common/sys/log.c

```diff
@@ -218,6 +227,14 @@ int log_var_register(int idx, char *name, void *addr, uint32_t samples_per_sec,
     vars[idx].type = type;
 
     // Calculate 'log_interval_usec' from samples per second
+    //
+    // This calculation of log_interval_usec may result in the integer truncation of
+    // a decimal value, ie for samples_per_sec = 3000,
+    // log_interval_usec = (1,000,000 / 3,000) = 333.333 which becomes 333 usec
+    //
+    // Because this integer truncation happens the same way here as it does in
+    // log_var_register(), we do not expect any samples to get missed when logging
+    // even when running at not-nice frequencies.
```

Comment:
Fix this comment, copy-paste issue :)

"...the same way here as it does in the `log_callback()`, we do not..."